kratos/python/add_logger_to_python.cpp

```diff
@@ -129,6 +133,7 @@ void  AddLoggerToPython(pybind11::module& m) {
     logger_scope.def_static("Print", printDefault); // raw_function(printDefault,1))
     logger_scope.def_static("PrintInfo",printInfo); // raw_function(printInfo,1))
     logger_scope.def_static("PrintWarning", printWarning); //raw_function(printWarning,1))
+    logger_scope.def_static("Flush", flush);
```

Comment:
Why do you need an auxiliary function for this?

Reply:
Yes, I think we can remove a couple of lines here. Will do.